### ISSUE TITLE
Case sensitivity

### ISSUE THREAD

**Issue by twsh** [sjpfenninger/citebibtex#3]
The parser preserves case of fields from the original bibtex file, but the `get_item()` function only looks for lower case fields. This is a problem because bibtex/biblatex aren't case sensitive, and various ways of automatically generating bib files do things with case differently: it's not a good general solution to just change the local copy to the right case although it's fine as a workaround.

I thought of a possible quick fix: Define a function that lowercases the keys of a dictionary and use that in the `update_refs()` function. Here's my attempt:
```
def update_refs(self, ref_file):
    with open(ref_file, 'r') as f:
        bib = Bibparser(f.read())
    bib.parse()
    refs = bib.records
    self.refs[ref_file] = [get_item(key_lower(refs[i])) for i in refs]
```

```
def key_lower(d):
    """ dict -> dict
    Return a dictionary identical to the original
    except that string keys are lower case.
    Assumes that the dictionary does not have keys
    that differ only in their case.
    """
    for k in d.keys():
        v = d.pop(k)
        if (type(k) is str) and (not k.islower()):
            k = k.lower()
        d[k] = v
    return d
```
It seems to work although I wouldn't be surprised if there are problems I'm not seeing.

**Comment by sjpfenninger** [comment]
Thanks for finding this! Indeed this is problematic. I'll see how this can best be addressed (possibly simply by switching parser library as you suggest in #5).

**Comment by sjpfenninger** [comment]
This is fixed in 956f523 as far as I can see thanks to switching to python-bibtexparser, let me know if any issues
